apiclient/account.go

```diff
@@ -260,20 +260,22 @@ func (c *HTTPclient) AccountSetMetadata(metadata *api.AccountMetadata) (types.He
 	return accv.TxHash, nil
 }
 
-// GetTransfers returns the list of token transfers associated with an account
-func (c *HTTPclient) GetTransfers(from common.Address, page int) ([]*indexertypes.TokenTransferMeta, error) {
+// GetTransfers returns the list of sent and received token transfers associated with an account
+func (c *HTTPclient) GetTransfers(from common.Address, page int) (map[string][]*indexertypes.TokenTransferMeta, error) {
```

Comment:
i'd rename `from` to something less misleading, like `account`

Reply:
also i'd name the method ListTransfers or even ListTokenTransfers to be consistent with ListAccounts and CountTokenTransfers